tests/i18n/tests.py

```diff
@@ -2251,10 +2260,7 @@ def test_get_language_from_request(self):
 
     def test_get_language_from_request_null(self):
         lang = trans_null.get_language_from_request(None)
-        self.assertEqual(lang, "en")
-        with override_settings(LANGUAGE_CODE="de"):
-            lang = trans_null.get_language_from_request(None)
-            self.assertEqual(lang, "de")
```

Comment:
I think you should leave the setting in the test as it's part of what behaviour changes you'd requested in the ticket right? 🤔

ie make sure with setting "de" it still returns None…